### DISCUSSION TITLE
ChromeOS 106 and above not working

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#75] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
My school district automatically updates our Chromebooks whenever it is turned off. One day, it updated to ChromeOS 106. When I logged on my Chromebook I had my enforcement extensions back, I tried using it and it didn't work. Please fix?

**Comment by 3kh0** [comment, marked as answer]
There is no fix for that, it got patched in that version.

**Reply by ghost** [reply]
how do i make a html file on my chromebook

**Reply by 3kh0** [reply]
@ttvbeghhf Look it up
